tyrian/js/src/main/scala/tyrian/Sub.scala

```diff
@@ -84,7 +83,7 @@ object Sub:
       Observe(
         id,
         observable,
-        Kleisli(toMsg).map(f).run
+        toMsg.andThen(_.map(f))
```

Comment:
I'm not going to kick up a fuss over it, but I'm finding that unattractive. :sweat_smile:

Next time, maybe: `toMsg andThen { _.map(f) }`